e2e-tests/security-context/run

```diff
@@ -11,9 +11,14 @@ deploy_cert_manager
 
 kubectl_bin apply -f "$test_dir/conf/service-account.yml"
 if [[ "${OPENSHIFT}" == 1 ]]; then
-    oc adm policy add-scc-to-user privileged -z percona-xtradb-cluster-operator ${OPERATOR_NS:+-n $OPERATOR_NS}
+    oc adm policy add-scc-to-user privileged -z default
```

Comment:
please remove if this line is not needed

Reply:
done